src/systems/filecoin_mining/storage_proving/post_generator.md

```diff
@@ -1,92 +0,0 @@
----
-title: PoSt Generator
```

Comment:
- not sure it makes sense to remove this component. this is meant to be a clear explanation of how the `StorageProving` subsystem creates posts, submits them etc. 
- The EC work should not talk about PoSt Submission. SPC can mention what's needed. We need a component (part of storage proving) that describes how (in detail) an implementation needs to produce a post.
- i would expect most of the Election-PoSt prose to land here

Reply:
Agreed, the large issue was mostly that this md was a full dup of the `storage_mining/mining_cycle.md` and content was better suited there. Will recreate a `post_generator` for the `Election-PoSt`